libs/api_common/api_common.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <map>
+#include <stdexcept>
+#include "ebpf_api.h"
+#include "ebpf_execution_context.h"
+#undef VOID
+#include "ebpf_helpers.h"
+#include "platform.hpp"
+extern "C"
+{
+#include "ubpf.h"
+}
+
+typedef struct _map_cache
+{
+    uintptr_t handle;
+    EbpfMapDescriptor ebpf_map_descriptor;
+} map_cache_t;
+
+typedef struct _ebpf_maps_section_record_windows
+{
+    uint32_t size;
+    uint32_t type;
+    uint32_t key_size;
+    uint32_t value_size;
+    uint32_t max_entries;
+} ebpf_maps_section_record_windows;
```

Comment:
Line 13 includes ebpf_helpers.h which already has:
```
typedef struct _ebpf_map_definition
{
    uint32_t size;
    uint32_t type;
    uint32_t key_size;
    uint32_t value_size;
    uint32_t max_entries;
} ebpf_map_definition_t;
```
So this struct seems to be an unnecessary duplicate (and doesn't end in _t which would be moot if this is deleted)

Reply:
Removed ebpf_maps_section_record_windows